contracts/oracles/UniswapV3PriceFeed.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@chainlink/contracts/src/v0.8/interfaces/FeedRegistryInterface.sol";
+import "@chainlink/contracts/src/v0.8/Denominations.sol";
+import "../interfaces/IMultipriceOracle.sol";
+import "../interfaces/IPriceFeed.sol";
+
+contract UniswapV3PriceFeed is IPriceFeed {
```

Comment:
Should we name this as WETH pair related?

Reply:
I would like to keep the name clean.